source/Calamari.Common/Features/StructuredVariables/YamlFormatVariableReplacer.cs

```diff
@@ -10,17 +10,18 @@
 
 namespace Calamari.Common.Features.StructuredVariables
 {
-    public interface IYamlFormatVariableReplacer : IFileFormatVariableReplacer
-    {
-    }
-
-    public class YamlFormatVariableReplacer : IYamlFormatVariableReplacer
+    public class YamlFormatVariableReplacer : IFileFormatVariableReplacer
```

Comment:
The changes to this file are minor, but the diff looks pretty gnarly in github. The changes should be:

- `IYamlFormatVariableReplacer` removed
- try/catch removed
- `bool TryModifyFile` => `void ModifyFile` (and changes to associated `returns`)
- `IsBestReplacerForFileName` added